docs/userguide/campaigns.md

```diff
@@ -1,3 +1,25 @@
 # Campaigns
```

Comment:
I think this PR is outdated and need a rebase, but I dont have much to complain about. I have some several additions Id like to make but will extend the user guide instructions at a later point